From 31cb17bb67d2af4d944f259c7b9928e9349e31e3 Mon Sep 17 00:00:00 2001
From: Nate Drude <nate.d@variscite.com>
Date: Mon, 14 Sep 2020 15:10:16 -0500
Subject: [PATCH] iMX8M: change DDR DMEM padding

Following U-boot's change 4332fdd (Change padding of DDR4 and LPDDR4 DMEM)
align the padding of DDR DMEM fw in imx-mkimage as well.

From meta-variscite-fslc ad930227.

Signed-off-by: Nate Drude <nate.d@variscite.com>
Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>
---
 iMX8M/soc.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index edcc397..b03455e 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -97,7 +97,7 @@ u-boot-spl-combine:
 
 u-boot-spl-ddr.bin: u-boot-spl.bin $(lpddr4_imem_1d) $(lpddr4_dmem_1d) $(lpddr4_imem_2d) $(lpddr4_dmem_2d)
 	@objcopy -I binary -O binary --pad-to 0x8000 --gap-fill=0x0 $(lpddr4_imem_1d) lpddr4_pmu_train_1d_imem_pad.bin
-	@objcopy -I binary -O binary --pad-to 0x4000 --gap-fill=0x0 $(lpddr4_dmem_1d) lpddr4_pmu_train_1d_dmem_pad.bin
+	@objcopy -I binary -O binary --pad-to 0x1000 --gap-fill=0x0 $(lpddr4_dmem_1d) lpddr4_pmu_train_1d_dmem_pad.bin
 	@objcopy -I binary -O binary --pad-to 0x8000 --gap-fill=0x0 $(lpddr4_imem_2d) lpddr4_pmu_train_2d_imem_pad.bin
 	@cat lpddr4_pmu_train_1d_imem_pad.bin lpddr4_pmu_train_1d_dmem_pad.bin > lpddr4_pmu_train_1d_fw.bin
 	@cat lpddr4_pmu_train_2d_imem_pad.bin $(lpddr4_dmem_2d) > lpddr4_pmu_train_2d_fw.bin
-- 
2.25.1

